### PR TITLE
puller

### PR DESCRIPTION
**Pull request by serapieTuyishime**
create it

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link href="./styles.css">
+</head>
+<body>
+    <h1>This is the first heading</h1>
```
Comment (Nkaka23dev) [inline index.html:11]: loremipsum

File: index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
```
Comment (Nkaka23dev) [inline index.html:4]: hsgegf

**Review by Nkaka23dev — CHANGES_REQUESTED** [review]
jsdgdghdf

File: index.html (hunk 2 above)
Comment (serapieTuyishime) [reply index.html:4]: not tis one

File: index.html (hunk 1 above)
Comment (serapieTuyishime) [reply index.html:11]: iu di this

**Pull request merged**